LLM_Compressor_0.9.0.md

```diff
@@ -0,0 +1,142 @@
+# LLM Compressor v0.9.0 release notes
+
+While working on this release, LLM Compressor reached over 100 contributors. Thank you to everyone for your contributions!
+
+This LLM Compressor v0.9.0 release adds:
+
+- Extended KV cache and attention quantization support
+- Batched calibration support
+- A new `model_free_ptq` pathway
+- A new _AutoRound_ modifier
+- Experimental support for MXFP4 quantization
+
+Additionally, this release finalizes the initial support work for SpinQuant R3 style transforms.
+This release also updates AWQ support to be further simplified and generic to quantization schemes beyond W4A16.
+
+<!-- https://issues.redhat.com/browse/INFERENG-2911 -->
+## Updated support for KV Cache and attention quantization ✨
+
+KV cache quantization has been refactored to address limitations in supported schemes and lifecycle-related bugs.
+The implementation has moved to compressed tensors, with new support for attention quantization.
+
+This refactoring enables the following updates:
+- Quantization of KV cache and attention using any scheme, including the new per-head strategy.
+- Running KV/attention quantization experiments within Hugging Face for research and accuracy validation.
+- Full compatibility with attention rotation via transforms.
+
+These changes lay the groundwork for future work in creating and researching advanced attention and KV-cache quantized models.
+
+<!-- https://issues.redhat.com/browse/INFERENG-3135 -->
+<!-- https://issues.redhat.com/browse/INFERENG-3136 -->
+## Added model-free post-training quantization ✨
+
+Model-free post-training quantization (PTQ) enables quantization by directly operating on safetensors files.
+This is particularly useful for models without a `transformers` model definition, such as some Mistral models.
```

Comment:
Can we add: 

This pathway is currently supported for data-free pathways only i.e FP8 quantization and was leveraged to quantize the [Mistral Large 3 model](https://huggingface.co/mistralai/Mistral-Large-3-675B-Instruct-2512).